### PR TITLE
Get All Topics

### PR DESCRIPTION
**Pull request by salaxander**
Still working on the PUT endpoint, but I figured I would throw this one up in the meantime. Thanks @tarebyte and @kytrinyx for the help getting started on this. Virtual 🍻 to the both of you!

### CODE REVIEW

File: lib/octokit/client/repositories.rb [hunk 1]
```diff
@@ -480,9 +480,24 @@ def tags(repo, options = {})
...
+      # List all topics for a repo
       #
       # Requires authenticated client for private repos.
+      # @param repo [Integer, String, Hash, Repositroy] A GitHub repository.
+      # @return [Array<Sawyer::Resource>] Array of hashes representing topics.
+      # @see https://developer.github.com/v3/repos/#list-all-topics-for-a-repository
+      # @example
+      #   Octokit.topics('octokit/octokit.rb')
+      # @example
+      #   @client.topics('octokit/octokit.rb')
```
Comment (kytrinyx) [inline lib/octokit/client/repositories.rb]: Totally minor copy/pasta thing—the example is duplicated.

**Comment by kytrinyx** [comment]
This is looking great! ✨

File: lib/octokit/client/repositories.rb (hunk 1 above)
Comment (salaxander) [reply lib/octokit/client/repositories.rb]: Whoops! I'll update in a moment

**Comment by kytrinyx** [comment]
@salaxander Would you give us a ping here when this is ready for final review?

**Comment by tarebyte** [comment]
Gentle bump @salaxander are you still interested in finishing this?

**Comment by tarebyte** [comment]
Closing out due to staleness, please feel free to open this back up.

**Pull request closed without merging**